nexus_exporter.py

```diff
@@ -245,7 +255,12 @@ def _request_data(self):
         self._data = json.loads(urllib2.urlopen(data_request).read())
 
 
+def fatal(msg):
+    print(msg)
+    os._exit(1)  # hard exit without throwing exception
```

Comment:
why wouldn't you want the exception?

Reply:
seems to be some threading oddness and nothing catches the `SystemExit` exception, everything stays running.